content/news/release-20150412.md

```diff
@@ -0,0 +1,41 @@
+We are very pleased to announce the Spring release of OpenRA.  This is a huge step forward for the Dune 2000 mod in particular, but also, behind the scenes, for the Tiberian Sun mod which has received support for some crucial features during this cycle.
+
+This release includes a total of over 1400 changes by 53 contributors, the culmination of five and a half months of relentless work, and with almost 2000 files and more than 210,000 changed lines one of the busiest development cycles to date.
+
+The development focus has been on the Dune 2000 mod, which has made some huge steps forward and contains the most visible changes. A new sidebar, sandworms and carryalls are the big new features, which should make any Dune 2000 player feel right at home in our mod. The other mods were not neglected, though. In Red Alert, the standout feature is the support of playable factions such as Russia, England, France, Ukraine and Germany. In both Red Alert and Tiberian Dawn, new single-player missions, both campaign and stand-alone ones, are available that will keep you occupied.
+
+Of course, we have not only improved the mods themselves. Here is a list of changes that affect the game in general:
+
+* The server browser now shows games across all mods, allowing to join a game from a different mod than the one currently loaded.
+* It is now possible to disable the "short game" mode that has been the de-facto default in the past.
+* The player name and colour can now be set up in the settings dialog.
+* When creating a server, the game informs the host about port forwarding issues.
+* Left-click mouse orders have been markedly improved in this release.
+* Some missions now support difficulty levels.
+* A huge number of smaller gameplay fixes and improvements have been added. A few examples:
+   * After having been captured while attacking a target, defense structures will now stop firing at that target (if it's now friendly).
+   * Friendly units blocking the exit of a production building will now move out of the way if needed.
+   * Units will no longer hunt cloaked units.
+   * The spy plane in Red Alert will no longer lure your anti-air capable units across the map.
+   * Units that have been loaded into a transport will now be correctly removed from the minimap.
```

Comment:
I think there's a couple to many of these minor points - this one is probably the least important, and could be removed.